fedimint-api-client/Cargo.toml

```diff
@@ -38,6 +38,9 @@ tokio-rustls = { version = "0.26.0", default-features = false, features = ["logg
 webpki-roots = { version = "0.26.3" }
 rustls-pki-types = { version = "1.7.0" }
 arti-client = { version = "0.20.0", features = ["tokio", "rustls", "onion-service-client"], default-features = false }
+# We need to pin this arti's `curve25519-dalek` dependency, due to `https://rustsec.org/advisories/RUSTSEC-2024-0344` vulneraliblity
+# It's been updated by https://gitlab.torproject.org/tpo/core/arti/-/merge_requests/2211, should be removed in next release.
+curve25519-dalek = { version = ">=4.1.3" }
```

Comment:
@dpc any other specific way I should pin a dependency instead?

Reply:
`= "4.1.3"`, is the same as `= ">=4.1.3"`, IIRC, so this is not really "pinning" anything.

Also I don't think one can pin indirect dependency. The line here **will most probably** make dependency unification ensure this or newer version. @oleonardolima

Reply:
> [...] The line here will most probably make dependency unification ensure this or newer version [...]

Yes, that's what I'm looking for. Asked because I wasn't sure if it required some other Nix-related change. All good, then.